pkg/services/ngalert/schedule/schedule.go

```diff
@@ -373,21 +373,7 @@ func (sch *schedule) ruleRoutine(grafanaCtx context.Context, key ngmodels.AlertR
 		logger := logger.New("version", e.rule.Version, "attempt", attempt, "now", e.scheduledAt)
 		start := sch.clock.Now()
 
-		schedulerUser := &user.SignedInUser{
-			UserID:           -1,
-			IsServiceAccount: true,
-			Login:            "grafana_scheduler",
-			OrgID:            e.rule.OrgID,
-			OrgRole:          org.RoleAdmin,
-			Permissions: map[int64]map[string][]string{
-				e.rule.OrgID: {
-					datasources.ActionQuery: []string{
-						datasources.ScopeAll,
-					},
-				},
-			},
-		}
-		evalCtx := eval.Context(ctx, schedulerUser)
+		evalCtx := eval.NewContext(ctx, ScheduleUserFor(e.rule.OrgID))
```

Comment:
```suggestion
		evalCtx := eval.NewContext(ctx, SchedulerUserFor(e.rule.OrgID))
```
unbelievably tiny nitpick